apps/meteor/client/startup/startup.ts

```diff
@@ -60,31 +57,6 @@ Meteor.startup(() => {
 			fireGlobalEvent('status-changed', status);
 		}
 	});
-
-	Tracker.autorun(async (c) => {
-		const uid = Meteor.userId();
-		if (!uid) {
-			return;
-		}
-
-		if (!hasPermission('manage-cloud')) {
-			return;
-		}
-
-		const {
-			registrationStatus: { workspaceRegistered },
-		} = await sdk.rest.get('/v1/cloud.registrationStatus');
-		c.stop();
-
-		if (workspaceRegistered !== true) {
-			banners.open({
-				id: 'cloud-registration',
-				title: () => t('Cloud_registration_pending_title'),
-				html: () => t('Cloud_registration_pending_html'),
```

Comment:
The associated translations must be removed too, as they don't seem to be used elsewhere.